docs/python-examples/fixtures/conftest.py

```diff
@@ -0,0 +1,109 @@
+import logging
+
+import pytest
+
+import ocs
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def storage_class(request):
+    """
+    Storage class fixture
+
+    Returns:
+        StorageClass: object of storage class
+    """
+    sc_name = ocs.get_random_name('storage_class')
+    sc = ocs.StorageClass(sc_name)
+    logger.info(f"Creating storage class: {sc.name}")
+
+    def fin():
+        if not sc.is_deleted:
+            sc.delete()
+
+    request.addfinalizer(fin)
+    return sc_name
+
+
+@pytest.fixture(scope='class')
+def cls_pvc(request, storage_class):
+    """
+    PVC fixture
+
+    Returns:
+        PVC: object of pvc class
+    """
+    pvc_name = ocs.get_random_name('pvc')
+    pvc = ocs.PVC(pvc_name, storage_class)
+    logger.info(f"Creating pvc: {pvc.name}")
+
+    def fin():
+        if not pvc.is_deleted:
+            pvc.delete()
+
+    request.addfinalizer(fin)
+    return pvc
+
+
+@pytest.fixture(scope='class')
+def pvcs_factory(request, storage_class):
```

Comment:
Would a naming convention of `create_{resource}` be better for factories instead of `{resource}_factory`? When a test actually uses the function I think it makes more sense for it to be `pvc = create_pvc(storage_class)` than `pvc = pvc_factory(storage_class)`

Reply:
Sure

Reply:
done